routers/api/v1/admin/cron.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package admin
+
+import (
+	"net/http"
+
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/cron"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/structs"
+	"code.gitea.io/gitea/routers/api/v1/utils"
+)
+
+// GetCron api for getting cron tasks
+func GetCron(ctx *context.APIContext) {
```

Comment:
one thing but this is no blocker: instead of "GetCron" you could rename it to ListCronTasks to be more acurate ...

Reply:
```suggestion
func ListCronTasks(ctx *context.APIContext) {
```